hedera-node/src/main/java/com/hedera/services/txns/contract/operation/HederaCreateOperation.java

```diff
@@ -52,8 +52,14 @@ public Gas cost(final MessageFrame frame) {
 
 		final Gas memoryGasCost = gasCalculator().memoryExpansionGasCost(frame, initCodeOffset, initCodeLength);
 
+		WorldStateAccount hederaAccount =
+				((HederaWorldUpdater) frame.getWorldUpdater()).getHederaAccount(frame.getRecipientAddress());
+		if (hederaAccount == null) {
+			hederaAccount = ((HederaWorldUpdater) frame.getWorldUpdater()).getHederaAccount(frame.getSenderAddress());
+		}
```

Comment:
I think that you might have not merged the latest changes from the `ethereumj-besu-swap` branch

Reply:
Yep, not merged to the most recent.